src/components/MultiStepIndicator/__tests__/multiStepIndicator-snapshot-tests.jest.tsx

```diff
@@ -23,7 +23,7 @@ const exampleSteps = [
 
 describe("MultiStepIndicator renders correctly", () => {
   it("with empty props", () => {
-    const tree = renderer.create(<MultiStepIndicator />).toJSON();
+    const tree = renderer.create(<MultiStepIndicator steps={[]} />).toJSON();
```

Comment:
If steps can be rendered with empty array why steps is required?

Reply:
exactly for the reason mentioned above, to not break/change existing API.
you want to change it anyway to be optional?

Reply:
before my change:
```
steps: PropTypes.arrayOf(
  PropTypes.shape({
    titleText: PropTypes.string,
    subtitleText: PropTypes.string,
    status: PropTypes.oneOf([
      MultiStepIndicator.stepStatuses.PENDING,
      MultiStepIndicator.stepStatuses.ACTIVE,
      MultiStepIndicator.stepStatuses.FULFILLED
    ])
  })
).isRequired,
```

Reply:
Change API for steps to be optional